### ISSUE TITLE
use conda environment in docs build

### ISSUE THREAD

**Issue by raybellwaves** [anitagraser/movingpandas#191]
docs are failing in https://github.com/anitagraser/movingpandas/pull/190 (see https://readthedocs.org/projects/movingpandas/builds/16235093/).

The issue looks like a virtualenv/catopy error.

The docs could be build from a conda environment (see https://github.com/geopandas/geopandas/blob/main/doc/environment.yml)

Not sure where in geopandas that environment is being picked up i.e. may be on the rtd/sphinx side

**Comment by anitagraser** [comment]
https://stackoverflow.com/questions/69774313/why-does-rtd-run-python-env-create-instead-of-conda-env-create-when-building

**Comment by anitagraser** [comment]
#197 

```
[rtd-command-info] start-time: 2022-03-05T13:33:08.032744Z, end-time: 2022-03-05T13:42:03.148696Z, duration: 535, exit-code: 137
conda env create --quiet --name 197 --file docs/environment.yml
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... Killed


Command killed due to timeout or excessive memory consumption
```

**Comment by anitagraser** [comment]
fixed in  #197. Thanks for the tip to copy geopandas, Ray.
